pkg/roachprod/install/cluster_synced.go

```diff
@@ -1195,6 +1225,33 @@ func formatProgress(p float64) string {
 	return fmt.Sprintf("[%s%s] %.0f%%", progressDone[i:], progressTodo[:i], 100*p)
 }
 
+// PutString into the specified file on the remote(s).
```

Comment:
Looking at the uses here, we seem to only want to do this on a single node (usually the one running prom+grafana). Lets update this signature to explicitly take in the set of nodes it wants to deal with instead of forcing callers to construct a `SyncedCluster` with just the set of nodes of interest (all the `promC` stuff). `c.Put` below will also need updating. This is present in the diff linked above.